### ISSUE TITLE
triplet loss

### ISSUE THREAD

**Issue by Phoebe-star** [davidsandberg/facenet#547]
why  does  triplet loss not do  tf. sqrt       , only do tf.square
  pos_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, positive)), 1)
        neg_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, negative)), 1)

**Comment by danielkaifeng** [comment]
I am also thinking of this topic, as by sqrt after square is euclidiean distance.
I tried to use abs() and this metric end up in in a little bit poor result.
So I guess tf.square is better after their experiment.
